modules/ROOT/pages/setting-up-your-development-environment.adoc

```diff
@@ -64,7 +64,7 @@ This process only needs to be done once. A project manager or whoever is respons
 
 === Enable Private Cloud Edition Setup
 
-https://archive.docs.mulesoft.com/release-notes/studio/anypoint-studio-6.2-with-3.8.3-runtime-release-notes[Anypoint Studio 6.2] and higher support the xref:private-cloud::index.adoc[Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)] version 1.7 and earlier.
+Anypoint Studio 6.2 and higher support the xref:private-cloud::index.adoc[Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)] version 1.7 and earlier.
```

Comment:
```suggestion
Anypoint Studio versions 6.2 and later support the xref:private-cloud::index.adoc[Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)] version 1.7 and earlier.
```

When referring to release spans, use "earlier" and "later" rather than "lower" and "higher" or "below" and "above."